ecbuff.h

```diff
@@ -54,6 +54,11 @@ ECB_VOID_BOOL_T ecbuff_read(ecbuff* const restrict rb, void* const restrict elem
 ECB_UINT_T ecbuff_unused(const ecbuff* const restrict rb);
 ECB_UINT_T ecbuff_used(const ecbuff* const restrict rb);
 
+/* ecbuff_reset
+ * Resets the buffer
+ */
+void ecbuff_reset(ecbuff* const restrict rb);
```

Comment:
```suggestion
void ecbuff_clear(ecbuff* const restrict rb);
```
I think I'd prefer clear, as this is what's used for C++ containers.